examples/common_patterns_for_connectors/priority_first_sync_for_high_volume_initial_syncs/mock_api.py

```diff
@@ -0,0 +1,40 @@
+
+from fivetran_connector_sdk import Logging as log
+import random
+import string
+from datetime import datetime, timedelta, timezone
+
+import connector
+
+def get_mock_api_response(base_url, params):
+    log.info(f"Making API call to url: {base_url} with params: {params}")
+    updated_since = connector.get_datetime_object(params['updated_since'])
+    until = updated_since + timedelta(days=1)
+    if (until > datetime.now(timezone.utc)):
+        until = datetime.now(timezone.utc)
+    response_page = {
+        "data": []
+    }
+    while updated_since <= until:
+        response_page['data'].append(
+            {
+                "id": string_generator(),
+                "name": string_generator(chars=string.ascii_lowercase),
+                "email": string_generator(chars=string.ascii_lowercase),
+                "address": string_generator(chars=string.ascii_lowercase),
+                "company": string_generator(chars=string.ascii_lowercase),
+                "job": string_generator(chars=string.ascii_lowercase),
+                "updated_at": updated_since.isoformat(),
+                "created_at": updated_since.isoformat(),
```

Comment:
nit: you can use faker https://pypi.org/project/Faker/
and mention it in requirements.txt to generate better data

Reply:
Thanks for pointing.